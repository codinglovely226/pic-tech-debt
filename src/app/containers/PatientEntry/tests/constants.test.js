import {
  REPLACE_PATIENT_MEDICAL_RECORD_NUMBER,
  REPLACE_PATIENT_NAME,
  UPDATE_PATIENT_AGE,
  UPDATE_PATIENT_AGE_GROUP,
  UPDATE_PATIENT_ANIMAL_TYPE,
  UPDATE_PATIENT_CATEGORY,
  UPDATE_PATIENT_CIRCUMSTANCE,
  UPDATE_PATIENT_LOCATION,
  UPDATE_PATIENT_MEDICAL_RECORD_NUMBER,
  UPDATE_PATIENT_NAME,
  UPDATE_PATIENT_OCCUPATION,
  UPDATE_PATIENT_POISON_SEVERITY_SCORE,
  UPDATE_PATIENT_SUBURB,
  UPDATE_PATIENT_POSTCODE,
  UPDATE_PATIENT_GEOMETRY,
  UPDATE_PATIENT_PREGNANT_LACTATING,
  UPDATE_PATIENT_PREGNANT_STATUS,
  UPDATE_PATIENT_PREGNANT_TRIMESTER,
  UPDATE_PATIENT_PREGNANT_TRIMESTER_UNKNOWN,
  UPDATE_PATIENT_RISK_ASSESSMENT,
  UPDATE_PATIENT_SEX,
  UPDATE_PATIENT_WEIGHT,
} from '../constants';

describe('Patient action constants', () => {
  it('REPLACE_PATIENT_MEDICAL_RECORD_NUMBER', () => {
    expect(REPLACE_PATIENT_MEDICAL_RECORD_NUMBER).toMatchSnapshot();
  });
  it('REPLACE_PATIENT_NAME', () => {
    expect(REPLACE_PATIENT_NAME).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_AGE', () => {
    expect(UPDATE_PATIENT_AGE).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_AGE_GROUP', () => {
    expect(UPDATE_PATIENT_AGE_GROUP).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_ANIMAL_TYPE', () => {
    expect(UPDATE_PATIENT_ANIMAL_TYPE).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_CATEGORY', () => {
    expect(UPDATE_PATIENT_CATEGORY).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_CIRCUMSTANCE', () => {
    expect(UPDATE_PATIENT_CIRCUMSTANCE).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_LOCATION', () => {
    expect(UPDATE_PATIENT_LOCATION).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_MEDICAL_RECORD_NUMBER', () => {
    expect(UPDATE_PATIENT_MEDICAL_RECORD_NUMBER).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_NAME', () => {
    expect(UPDATE_PATIENT_NAME).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_OCCUPATION', () => {
    expect(UPDATE_PATIENT_OCCUPATION).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_POISON_SEVERITY_SCORE', () => {
    expect(UPDATE_PATIENT_POISON_SEVERITY_SCORE).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_SUBURB', () => {
    expect(UPDATE_PATIENT_SUBURB).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_POSTCODE', () => {
    expect(UPDATE_PATIENT_POSTCODE).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_GEOMETRY', () => {
    expect(UPDATE_PATIENT_GEOMETRY).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_PREGNANT_LACTATING', () => {
    expect(UPDATE_PATIENT_PREGNANT_LACTATING).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_PREGNANT_STATUS', () => {
    expect(UPDATE_PATIENT_PREGNANT_STATUS).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_PREGNANT_TRIMESTER', () => {
    expect(UPDATE_PATIENT_PREGNANT_TRIMESTER).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_PREGNANT_TRIMESTER_UNKNOWN', () => {
    expect(UPDATE_PATIENT_PREGNANT_TRIMESTER_UNKNOWN).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_RISK_ASSESSMENT', () => {
    expect(UPDATE_PATIENT_RISK_ASSESSMENT).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_SEX', () => {
    expect(UPDATE_PATIENT_SEX).toMatchSnapshot();
  });
  it('UPDATE_PATIENT_WEIGHT', () => {
    expect(UPDATE_PATIENT_WEIGHT).toMatchSnapshot();
  });
});
