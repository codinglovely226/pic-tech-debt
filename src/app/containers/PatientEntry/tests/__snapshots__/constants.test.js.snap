exports[`Patient action constants REPLACE_PATIENT_MEDICAL_RECORD_NUMBER 1`] = `"pic/PatientEntry/REPLACE_PATIENT_MEDICAL_RECORD_NUMBER"`;

exports[`Patient action constants REPLACE_PATIENT_NAME 1`] = `"pic/PatientEntry/REPLACE_PATIENT_NAME"`;

exports[`Patient action constants UPDATE_PATIENT_AGE 1`] = `"pic/PatientEntry/UPDATE_PATIENT_AGE"`;

exports[`Patient action constants UPDATE_PATIENT_AGE_GROUP 1`] = `"pic/PatientEntry/UPDATE_PATIENT_AGE_GROUP"`;

exports[`Patient action constants UPDATE_PATIENT_ANIMAL_TYPE 1`] = `"pic/PatientEntry/UPDATE_PATIENT_ANIMAL_TYPE"`;

exports[`Patient action constants UPDATE_PATIENT_CATEGORY 1`] = `"pic/PatientEntry/UPDATE_PATIENT_CATEGORY"`;

exports[`Patient action constants UPDATE_PATIENT_CIRCUMSTANCE 1`] = `"pic/PatientEntry/UPDATE_PATIENT_CIRCUMSTANCE"`;

exports[`Patient action constants UPDATE_PATIENT_GEOMETRY 1`] = `"pic/PatientEntry/UPDATE_PATIENT_GEOMETRY"`;

exports[`Patient action constants UPDATE_PATIENT_LOCATION 1`] = `"pic/PatientEntry/UPDATE_PATIENT_LOCATION"`;

exports[`Patient action constants UPDATE_PATIENT_MEDICAL_RECORD_NUMBER 1`] = `"pic/PatientEntry/UPDATE_PATIENT_MEDICAL_RECORD_NUMBER"`;

exports[`Patient action constants UPDATE_PATIENT_NAME 1`] = `"pic/PatientEntry/UPDATE_PATIENT_NAME"`;

exports[`Patient action constants UPDATE_PATIENT_OCCUPATION 1`] = `"pic/PatientEntry/UPDATE_PATIENT_OCCUPATION"`;

exports[`Patient action constants UPDATE_PATIENT_POISON_SEVERITY_SCORE 1`] = `"pic/PatientEntry/UPDATE_PATIENT_POISON_SEVERITY_SCORE"`;

exports[`Patient action constants UPDATE_PATIENT_POSTCODE 1`] = `"pic/PatientEntry/UPDATE_PATIENT_POSTCODE"`;

exports[`Patient action constants UPDATE_PATIENT_PREGNANT_LACTATING 1`] = `"pic/PatientEntry/UPDATE_PATIENT_PREGNANT_LACTATING"`;

exports[`Patient action constants UPDATE_PATIENT_PREGNANT_STATUS 1`] = `"pic/PatientEntry/UPDATE_PATIENT_PREGNANT_STATUS"`;

exports[`Patient action constants UPDATE_PATIENT_PREGNANT_TRIMESTER 1`] = `"pic/PatientEntry/UPDATE_PATIENT_PREGNANT_TRIMESTER"`;

exports[`Patient action constants UPDATE_PATIENT_PREGNANT_TRIMESTER_UNKNOWN 1`] = `"pic/PatientEntry/UPDATE_PATIENT_PREGNANT_TRIMESTER_UNKNOWN"`;

exports[`Patient action constants UPDATE_PATIENT_RISK_ASSESSMENT 1`] = `"pic/PatientEntry/UPDATE_PATIENT_RISK_ASSESSMENT"`;

exports[`Patient action constants UPDATE_PATIENT_SEX 1`] = `"pic/PatientEntry/UPDATE_PATIENT_SEX"`;

exports[`Patient action constants UPDATE_PATIENT_SUBURB 1`] = `"pic/PatientEntry/UPDATE_PATIENT_SUBURB"`;

exports[`Patient action constants UPDATE_PATIENT_WEIGHT 1`] = `"pic/PatientEntry/UPDATE_PATIENT_WEIGHT"`;
